servers/rendering/rendering_server_default.cpp

```diff
@@ -379,6 +379,7 @@ void RenderingServerDefault::sync() {
 }
 
 void RenderingServerDefault::draw(bool p_swap_buffers, double frame_step) {
+	ERR_FAIL_COND_MSG(!::Thread::is_main_thread(), "Manually triggering the draw function from the RenderingServer can only be done on the main thread. Call this function from the main thread or use call_deferred().");
```

Comment:
Why the extra scope of `::Thread`?

Reply:
Just copy pasted from other places :) I will remove it.